routes/api.php

```diff
@@ -14,6 +14,13 @@
 Route::put('/managers/{id}', 'ManagerController@update')->name('managers.update');
 Route::delete('/managers/{id}', 'ManagerController@destroy')->name('managers.destroy');
 
+Route::post('/managers/register', 'ManagerController@register')->name('managers.register');
```

Comment:
Registering a manger should be `POST /managers`. 'Registering' is the same as creating, right?

Reply:
In this case it was separated to have all the CRUD calls by default and special routes secondary. Having specific register/login calls made it easier to test and build without touching the original CRUD functionality.